nexus/reconfigurator/planning/src/system.rs

```diff
@@ -596,6 +633,43 @@ impl SystemDescription {
                     part_number: sp_state.model.clone(),
                     serial_number: sp_state.serial_number.clone(),
                 };
+                if let Some(stage0) = &s.stage0_caboose() {
```

Comment:
Is the `&` required here and in `stage0_next_caboose` below?

Reply:
aha! it is not 😄